packages/tools/src/eventListeners/mouse/mouseDownListener.ts

```diff
@@ -9,6 +9,25 @@ import getMouseEventPoints from './getMouseEventPoints';
 const { MOUSE_DOWN, MOUSE_DOWN_ACTIVATE, MOUSE_CLICK, MOUSE_UP, MOUSE_DRAG } =
   Events;
 
+// The amount of time in milliseconds within which a browser 'dblclick' event has to occur.
+// Any mouse down, up, down and up sequence taking longer than this time is considered to
+// NOT be a double click and any browser 'dblclick' event that subsequently occurs as a result
+// of such a sequence will be ignored. It is best to set this to a value that is less
+// than the system value for detecting a double click. Setting something too large
+// might detect a double click that does not constitute a browser 'dblclick' and thus
+// no mouse events for the sequence will get fired at all.
+//
+// TODO This module should detect and fire 'dblclick' events at its discretion and
+// ignore all those generated by the browser.
+//
+const DOUBLE_CLICK_TOLERANCE_MS = 400;
+
+// A drag (distance) during the double click timeout that is greater than this
+// value will cancel the timeout and suppress any double click that might occur.
+// This tolerance is particularly important on touch devices where some movement
+// might occur between the two clicks.
+const DOUBLE_CLICK_DRAG_TOLERANCE = 3;
```

Comment:
The last commit had this as 2. I did some further testing on my phone and thought 3 was better. I will add a TODO to revisit this value for touch devices.